src/install/default.txt

```diff
@@ -586,7 +586,7 @@ main() {
   case "${croc_arch}" in
      "x86_64" ) croc_arch="64bit";;
       "amd64" ) croc_arch="64bit";;
-    "aarch64" ) croc_arch="ARM64";;
```

Comment:
aarch64 is not a typo I believe

Reply:
I have revised the changes and squashed it as one single commit.